README.md

```diff
@@ -9,13 +9,42 @@ This module requires the following modules/libraries:
 * [Islandora](https://github.com/islandora/islandora)
 * [Islandora Scholar](https://github.com/islandora/islandora_scholar)
 
-## Installation
+## Installation & Configuration
 
 Install as usual, see [this](https://drupal.org/documentation/install/modules-themes/modules-7) for further information.
 
-## Configuration
-
-* Enable the module via Administration » Modules (admin/modules)
+First enable the "Islandora Scholar Extended Profiles" module, then subsequently (if you want to import the default NCES research classification schema) enable the "Islandora Scholar Extended Profile -- Activities Taxonomy Sample Data" submodule. This has other module dependencies which you will need to temporarily install and enable in order to import the hierarchical CSV file of research interests.
+
+Your site's Taxonomy list (admin/structure/taxonomy) will now include a new "Islandora Profile Activities" Taxonomy. It is empty, so we need to create it by hand, or import the provided sample taxonomy:
+* Install and enable all "Activities Taxonomy Sample Data" submodule dependencies: typically the two not already installed on an Islandora site are [Taxonomy_csv](https://www.drupal.org/project/taxonomy_csv) (if you have import errors see this [patch](https://www.drupal.org/project/taxonomy_csv/issues/2833513)) and [Term_ref_autocomplete](https://www.drupal.org/project/Term_ref_autocomplete)
+* Enable the "Activities Taxonomy Sample Data" submodule.
+* Download a local copy of the [NCES taxonomy](https://raw.githubusercontent.com/Born-Digital-US/islandora_scholar_profiles/master/modules/islandora_scholar_profile_activity_import/files/LASIR%20Research%20Classification%20Taxonomy%20-%20H_LASIR_Taxonomy_strict_levels.csv) and modify if desired.
```

Comment:
Is it intentional to have the CSV stored in Born-Digital's account? Perhaps more importantly, will this CSV persist in the BD repo?

Reply:
Good catch- revised doc below with ICG URL

Reply:
Good solution. Thanks!